### PR TITLE
[VEN-2770] Revoke Guardian Permissions

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: vips/vip-355/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ethereum.REDSTONE_ORACLE, "setTokenConfig(TokenConfig)", ethereum.GUARDIAN],
+        dstChainId: LzChainId.ethereum,
+      },
+      {
+        target: ETHEREUM_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ethereum.REDSTONE_ORACLE, "setDirectPrice(address,uint256)", ethereum.GUARDIAN],
+        dstChainId: LzChainId.ethereum,
+      },
```
Comment (chechu) [inline vips/vip-355/bscmainnet.ts:186]: I wouldn't revoke these permissions

File: vips/vip-356/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    version: "v2",
+    title: "VIP-356",
+    description: `### Description`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      // Revoke Permissions
```
Comment (chechu) [inline vips/vip-356/bscmainnet.ts:35]: Permissions on RewardDistributor contracts were set using the wildcard `0x0` (see it here, for [Arbitrum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/arbitrumone/vip-004/index.ts#L104-L113) and [Ethereum](https://github.com/VenusProtocol/vips/blob/main/multisig/proposals/ethereum/vip-002/index.ts#L155-L164)). So, I suppose we should revoke the permission on the `0x0` address, not on specific addresses

File: vips/vip-357/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    version: "v2",
+    title: "VIP-357",
+    description: `### Description`,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+  return makeProposal(
+    [
+      // Revoke Permissions
```
Comment (chechu) [inline vips/vip-357/bscmainnet.ts:50]: The zero address was used to grant permissions on the IR models. So, I suppose we shouldn't use specific addresses to revoke these permissions, but the zero address

File: vips/vip-356/bscmainnet.ts [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+          "updateJumpRateModel(uint256,uint256,uint256,uint256)",
+          opbnbmainnet.GUARDIAN,
+        ],
+        dstChainId: LzChainId.opbnbmainnet,
+      },
+
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PRIME, "setTokensDistributionSpeed(address[],uint256[])", arbitrumone.GUARDIAN],
```
Comment (GitGuru7) [inline vips/vip-356/bscmainnet.ts:129]:
```suggestion
        params: [ARBITRUM_ONE_PLP, "setTokensDistributionSpeed(address[],uint256[])", arbitrumone.GUARDIAN],
```

File: vips/vip-356/bscmainnet.ts [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PRIME, "setMaxTokensDistributionSpeed(address[],uint256[])", arbitrumone.GUARDIAN],
```
Comment (GitGuru7) [inline vips/vip-356/bscmainnet.ts:135]:
```suggestion
        params: [ARBITRUM_ONE_PLP, "setMaxTokensDistributionSpeed(address[],uint256[])", arbitrumone.GUARDIAN],
```

File: vips/vip-356/bscmainnet.ts [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PRIME, "setMaxLoopsLimit(uint256)", arbitrumone.GUARDIAN],
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PLP, "updateAlpha(uint128,uint128)", arbitrumone.GUARDIAN],
```
Comment (GitGuru7) [inline vips/vip-356/bscmainnet.ts:147]:
```suggestion
        params: [ARBITRUM_ONE_PRIME, "updateAlpha(uint128,uint128)", arbitrumone.GUARDIAN],
```

File: vips/vip-356/bscmainnet.ts [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PLP, "updateMultipliers(address,uint256,uint256)", arbitrumone.GUARDIAN],
```
Comment (GitGuru7) [inline vips/vip-356/bscmainnet.ts:153]:
```suggestion
        params: [ARBITRUM_ONE_PRIME, "updateMultipliers(address,uint256,uint256)", arbitrumone.GUARDIAN],
```

File: vips/vip-356/bscmainnet.ts [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+        dstChainId: LzChainId.arbitrumone,
+      },
+      {
+        target: ARBITRUM_ONE_ACM,
+        signature: "revokeCallPermission(address,string,address)",
+        params: [ARBITRUM_ONE_PLP, "setStakedAt(address[],uint256[])", arbitrumone.GUARDIAN],
```
Comment (GitGuru7) [inline vips/vip-356/bscmainnet.ts:159]:
```suggestion
        params: [ARBITRUM_ONE_PRIME, "setStakedAt(address[],uint256[])", arbitrumone.GUARDIAN],
```

Comment (GitGuru7) [reply vips/vip-356/bscmainnet.ts:159]: PLP should be replaced by prime and vice versa on many places

**Pull request closed without merging**